protos/flow.proto

```diff
@@ -167,6 +167,8 @@ message CreateRawTableOutput { string table_identifier = 1; }
 message TableSchema {
   string table_identifier = 1;
   repeated string primary_key_columns = 2;
+  // only used during SetupFlow, CDC fetches it everytime
+  // TODO: rename at some point
```

Comment:
can this be cleared out as migration so in future it can be removed?

Reply:
prefer not to remove it because I want entirety of SetupFlow to use the same value
want to just rename it to make sure it's not used elsewhere

Reply:
Can create model
```
type SetupTableSchema {
  *protos.TableSchema
  ...rest
}
```
& pass that around instead of the proto